packages/honeycomb-opentelemetry-web/README.md

```diff
@@ -201,6 +201,19 @@ Static fields are added to the [Resource](https://opentelemetry.io/docs/concepts
 
 Fields that can change during the lifetime of the page are instead added to each span in a [SpanProcessor](https://opentelemetry.io/docs/specs/otel/trace/sdk/#span-processor).
 
+#### GlobalErrorsInstrumentationConfig
```

Comment:
Seems like we have 2 README's that are the same, adding in the new info here too